deploy.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+echo "Running deploy script..."
+SOLANA_CONFIG=$1;
+PROGRAM_ID=$2;
+OWNER=$3;
+echo "Using Solana config filepath: $SOLANA_CONFIG"
+echo "Program ID: $PROGRAM_ID"
+echo "OWNER: $OWNER"
+
+solana config set --url https://api.devnet.solana.com
+
+SOURCE=`target/debug/spl-token --config $SOLANA_CONFIG wrap 21 2>&1 | awk '{print $NF}'`
```

Comment:
lets have end of script call unwrap

Reply:
What do you mean unwrap?

Reply:
also since this is really just for devnet lets only wrap 10 and ask for a 10 airdrop lol?